### ISSUE TITLE
Container.IsDisposed property not reflecting own scope disposed state.

### ISSUE THREAD

**Issue by A-Kjeldgaard** [dadhi/DryIoc#588]
If I open scopes within scopes (that is tracked by parent), and then disposes the outer/parent scope, then the inner scope is disposed as well.

However the `IsDisposed `property is still false, despite `OwnCurrentScope` being disposed.

```
//Arrange
var container = new Container();
var outerScope = container.OpenScope("Test", true);
var innerScope = outerScope.OpenScope("Test2", true);

//act
outerScope.Dispose();

//assert
Assert.True(innerScope.IsDisposed); // <-- Fails
```

**Comment by dadhi** [comment]
@A-Kjeldgaard Oh, interesting catch. I will check, thanks.

**Comment by dadhi** [comment]
fixed, will be out with v5.4.2
